### DISCUSSION TITLE
Tidal provider error

### DISCUSSION THREAD

**Discussion by gyurobenjamin** [music-assistant/support#2885] [Q&A] [answered]
Trying to set up Tidal, but with the very first button I'm getting this error:

```
AddProvider-d727396f.js:1 Uncaught TypeError: Cannot read properties of null (reading 'setAttribute')
    at AddProvider-d727396f.js:1:1301
```

**Comment by OzGav** [comment]
Any chance of some more details?

**Reply by gyurobenjamin** [reply]
Not sure what other information could be related here.

I'm running the container in kubernetes, but I think this is a client-side problem.

**Comment by OzGav** [comment]
Many people have reported problems with kubernetes but you can start here https://music-assistant.io/faq/troubleshooting/

**Reply by OzGav** [reply]
I should also add this from discord 

> Kubernetes is not supported by Home Assistant so it is also not supported by Music Assistant.

**Comment by gyurobenjamin** [comment, marked as answer]
Just pulled the beta docker, the issue was fixed there. Thanks
